example/pubspec.lock

```diff
@@ -7,7 +7,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.6.1"
+    version: "2.8.2"
```

Comment:
why changes inside `.lock` file, did you run `flutter upgrade`, not needed except if you are increasing dependecies versions